graphql/src/schema/api.rs

```diff
@@ -266,10 +265,22 @@ fn field_enum_filter_input_values(
 /// Generates `*_filter` input values for the given list field.
 fn field_list_filter_input_values(
     _schema: &Document,
-    _field: &Field,
-    _field_type: &Type,
-) -> Vec<InputValue> {
-    vec![]
+    field: &Field,
+    field_type: &Type,
+) -> Option<Vec<InputValue>> {
+    let _value_type = ast::get_field_value_type(field_type).ok()?;
+    Some(
+        vec!["", "not", "contains", "not_contains"]
+            .into_iter()
+            .map(|filter_type| {
+                input_value(
```

Comment:
This isn't quite right. For `contains` and `not_contains`, we'd want the input value to be a `T`, not a `[T]`.

This will affect the SQL for the two filter types as well.

Reply:
I changed my mind on this, this is fine for the moment.